dev-itpro/developer/devenv-application-overview.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Overview of the Application"
+description: "This article provides an overview of the modules in the Application, and information about how you can use them."
+author: bholtorf
+ms.topic: overview
+ms.author: bholtorf
+ms.date: 01/08/2024
+ms.collection:
+  - bap-ai-copilot
+ms.reviewer: bholtorf
+---
+
+# Overview of the Application
+
+[!INCLUDE[azure-ad-to-microsoft-entra-id](~/../shared-content/shared/azure-ad-to-microsoft-entra-id.md)]
+
+The business central application code is being refactored from a single monolythic application into smaller applications (layers) which in turn are built from small modules which handle a single functionality or feature.
+
+The System Application layer contains modules that provide ERP agnostic functionality. These are mostly tecnical features which any application needs.
+The Business Foundation layer contains modules that provide ERP related functionality that applies to all or none of the application domains.
```

Comment:
What about linking to the app code directly?

System Application: https://github.com/microsoft/BCApps/tree/main/src/System%20Application
Business Foundation: https://github.com/microsoft/BCApps/tree/main/src/Business%20Foundation

Reply:
done, thx :-)